prog/postgres/postgres_resource_nexus.rb

```diff
@@ -82,7 +88,9 @@ def before_run
     postgres_resource.server_cert, postgres_resource.server_cert_key = create_server_certificate
     postgres_resource.save_changes
 
-    hop_wait_server
+    reap
+    hop_wait_server if leaf?
+    nap 5
```

Comment:
If we nap here, does it not create certificates again and again?

Reply:
IMHO, the separate label is more suitable for it. But it's not a blocker.

Reply:
let's keep it as if for now then.